### PR TITLE
[IMP] buckaroo: provide steps to set up a webhook

### PR DESCRIPTION
**Pull request by AntoineVDV**
Here is what Buckaroo Plaza looks like for reference:
screenshot-plaza buckaroo nl-2022 01 21-15_11_27

task-2692127

See also:
- https://github.com/odoo/odoo/pull/82922

### CODE REVIEW

File: content/applications/general/payment_acquirers/buckaroo.rst [hunk 1]
```diff
@@ -5,8 +5,24 @@ Buckaroo
...
+
+Enable Push responses
+---------------------
+
+Log into `Buckaroo Plaza <https://plaza.buckaroo.nl>`_ and go to :menuselection:`My Buckaroo
+--> Websites --> Push settings`. Then,
+
+#. In the **Delayed and Push responses** section, check **Enable Push Response**.
+#. | In both the **Push URI Success** and **Push URI Failure** text fields, enter your Odoo database
+     URL followed by `/payment/buckaroo/webhook`.
```
Comment (LoredanaLrpz) [inline content/applications/general/payment_acquirers/buckaroo.rst:19]:
```suggestion
#. | Enter your Odoo database URL followed by `/payment/buckaroo/webhook` in both the **Push URI 
     Success** and **Push URI Failure** text fields
```

File: content/applications/general/payment_acquirers/buckaroo.rst [hunk 2]
```diff
@@ -5,8 +5,24 @@ Buckaroo
...
+Configuration on Buckaroo Plaza
+===============================
+
+Enable Push responses
+---------------------
+
+Log into `Buckaroo Plaza <https://plaza.buckaroo.nl>`_ and go to :menuselection:`My Buckaroo
+--> Websites --> Push settings`. Then,
+
+#. In the **Delayed and Push responses** section, check **Enable Push Response**.
```
Comment (LoredanaLrpz) [inline content/applications/general/payment_acquirers/buckaroo.rst:17]:
```suggestion
#. Check **Enable Push Response** in the **Delayed and Push responses** section.
```

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
I've made some suggestions to be more consistent in the way we tell "what to do" and then "where to do it" but overall it looks clear to me 

I'm still thinking of how to improve the general reading of these pages, working on it :)

**Review by LoredanaLrpz — APPROVED** [review]

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**